src/bindings/c/include/openvino/c/ov_remote_context.h

```diff
@@ -0,0 +1,162 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+/**
+ * @brief This is a header file for ov_remote_context C API
+ *
+ * @file ov_remote_context.h
+ */
+#pragma once
+#include "openvino/c/ov_common.h"
+#include "openvino/c/ov_shape.h"
+#include "openvino/c/ov_tensor.h"
+
+typedef struct ov_remote_context ov_remote_context_t;
+
+/**
+ * @brief remote context property key
+ */
+
+//!< Read-write property: shared device context type, can be either pure OpenCL (OCL) or
+//!< shared video decoder (VA_SHARED) context.
+//!< Value is string, it can be one of below strings:
+//!<    "OCL"       - Pure OpenCL context
+//!<    "VA_SHARED" - Context shared with a video decoding device
+OPENVINO_C_VAR(const char*) ov_property_key_intel_gpu_context_type;
+
+//!< Read-write property<void *>: identifies OpenCL context handle in a shared context or shared memory blob
+//!< parameter map.
+OPENVINO_C_VAR(const char*) ov_property_key_intel_gpu_ocl_context;
+
+//!< Read-write property<int string>: ID of device in OpenCL context if multiple devices are present in the context.
+OPENVINO_C_VAR(const char*) ov_property_key_intel_gpu_ocl_context_device_id;
+
+//!< Read-write property<int string>: In case of multi-tile system, this key identifies tile within given context.
+OPENVINO_C_VAR(const char*) ov_property_key_intel_gpu_tile_id;
+
+//!< Read-write property<void *>: OpenCL queue handle in a shared context
+OPENVINO_C_VAR(const char*) ov_property_key_intel_gpu_ocl_queue;
+
+//!< Read-write property<void *>: video acceleration device/display handle in a shared context or shared
+//!< memory blob parameter map.
+OPENVINO_C_VAR(const char*) ov_property_key_intel_gpu_va_device;
```

Comment:
Why do we have intel_gpu specific properties in common header? Can we have a plugin specific header with such variables?

Reply:
ok, let's have a specific header file for gpu device!